### DISCUSSION TITLE
哭了  2.0.9 添加了阿里网盘  下载 直接 报错了、、

### DISCUSSION THREAD

**Discussion by charlesouwen** [libsgh/PanIndex#225] [General]
InvalidArgument At least one of AdditionalHeaders is not in request headers. 61DF7899AC30323133EA06F5 bj29.cn-beijing.data.alicloudccp.com

这是咋了呀

**Comment by aiorobo** [comment]
我也是同样的问题只有阿里云盘这样报错

**Comment by libsgh** [comment]
挂载阿里云盘要用手机端token,获取方式看下文档
